plugins/flutter_aepconsent/lib/flutter_aepconsent.dart

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import 'dart:async';
+import 'package:flutter/services.dart';
+
+/// Adobe Experience Platform Consent API.
+class Consent {
+  static const MethodChannel _channel = 
+       const MethodChannel('flutter_aepconsent');
+
+  /// Gets the current AEPConsent extension version.
+  static Future<String> get extensionVersion =>
+      _channel.invokeMethod('extensionVersion').then((value) => value!);
+
+  /// Retrieves the current consent preferences stored in the Consent extension
+  static Future<Map<dynamic, dynamic>> get consents => _channel
```

Comment:
I am getting below error if I set to <Map<String, dynamic>> , using <Map<dynamic, dynamic>> seems the return comes back properly.
[VERBOSE-2:ui_dart_state.cc(209)] Unhandled Exception: type '_InternalLinkedHashMap<Object?, Object?>' is not a subtype of type 'Map<String, dynamic>?' in type cast
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:177:41)
<asynchronous suspension>
#1      _MyAppState.getConsent (package:flutter_aepsdk_example/main.dart:231:16)
<asynchronous suspension>

Reply:
The key type should be string here. This error may be caused due to the usage inside the flutter sample app - https://github.com/adobe/aepsdk_flutter/pull/1/files#diff-72fd825ad17ce7800a2078e06f0dff1e12836ee59760465e78d9a5d222cebd15R228

Reply:
Still looking into.

Reply:
Getting Map<object, object> response for getConsent.  Need to keep it to Map<String, dynamic> for this.

![ReturnInDart](https://user-images.githubusercontent.com/29806547/176952905-8e35f445-7de4-4d43-83a4-a17e24a87cf4.png)